tox.ini

```diff
@@ -0,0 +1,31 @@
+# To use tox, see https://tox.readthedocs.io
+# Simply pip or conda install tox
+# If you use conda, you may also want to install tox-conda
+# then run `tox` or `tox -- {pytest args}`
+# run in parallel using `tox -p`
+
+# see also test/rmq/docker-compose.yml to start a rabbitmq server, required for the those tests
+
+[tox]
+envlist = py37
+
+[testenv]
+usedevelop = true
+
+[testenv:py{36,37,38,39}]
```

Comment:
I don't think we support all of these versions. I would stick to 37 and 38.

Reply:
removed 36 👍